launcher/ui/GameListSelector.py

```diff
@@ -63,12 +62,16 @@ def populate_list(self):
         self.game_lists.insert(
             0, ["", gettext("Configs and Games")])
 
+        currIndex = self.get_index()
+        self.blockSignals(True)
```

Comment:
Since `self.clear()` unset the list items but also (of course) the current index, we need to save and restore after the list has been populated.

`blockSignals()` is needed because we are programmatically rewriting the list, therefore no signal should propagate. I didn't understand exactly which signal was triggered but following the code something was happening that reinitialized the object.